apps/artwork/components/related_artists/index.coffee

```diff
@@ -31,6 +31,7 @@ module.exports = ->
   relatedArtists = sd.CLIENT.artists[0].artists
   showMore = relatedArtists.length > 15
   $el = $('.js-artwork-artist-related-rail__content')
+  relatedArtists = _.take(relatedArtists, 15)
```

Comment:
No you are right, there probably should be a spec to cover this. The gist is that they should always advance by 4 and not breaking the order or the artist cells.